core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -54,8 +58,14 @@
  * to store metadata and manifests.
  */
 public class HadoopTables implements Tables, Configurable {
+
+  public static final String LOCK_PROPERTY_PREFIX = "iceberg.tables.hadoop.";
+
```

Comment:
Can you make this private and remove the newline? I don't think there's a reason for this prefix to be public.

Reply:
this is actually to address the comments at https://github.com/apache/iceberg/pull/3663#discussion_r774119018 from  @jackye1995

Reply:
That's okay, then.